.golangci.yml

```diff
@@ -32,13 +32,15 @@ linters:
     - goprintffuncname
     - gosec
     - gosimple
+    - gosmopolitan
```

Comment:
New linter for i18n checks.

Reply:
Where do we have i18n logic?

Reply:
We don't but apparently this linter helps to catch bad patterns that prevent adding it later.